pageserver/src/tenant/config.rs

```diff
@@ -278,6 +287,7 @@ impl Default for TenantConf {
                 DEFAULT_EVICTIONS_LOW_RESIDENCE_DURATION_METRIC_THRESHOLD,
             )
             .expect("cannot parse default evictions_low_residence_duration_metric_threshold"),
+            forced_image_creation_limit: DEFAULT_FORCED_IMAGE_CREATION_LIMIT,
```

Comment:
I think this makes the limit of 1024 applied by default. So it's an opt-out feature flag for the feature instead of opt-in, what I understand was the plan.

Reply:
I do not think that we should completely disable GC feedback by default.
Actually negative effect (storage size explosion) is caused mostly by switching on this feedback for existed projects with "stairs". If feedback is enabled fro the very beginning, then there should be not such explosion an storage size is expected to be smaller than without this feedback.

This is why I do not want to disable GC feedback by default and only switch it on manually for some particuar customers. I afraid that in this case this feedback will not be used at all. It means that we still have problems with "stairs". I am not sure if this GC feed back is the best way to address this problem, but untill we have implements some better alternative I think that we should try this one.

There is one more motivation for this GC feedback, rather than just reducing storage size caused by delta layers stairs: reduce number of page reconstruction. Image layers is the only way now to eliminate page reconstruction. Page cache is small and rconstructed pages are fast evicted fro it, because it used for all IO operations with disk. So I prefer to have some other tie based criteria for image layers generation except just counting delta layers since last image layer, because right now image layer generation can be delayed for arbitrary long time.  GC feedback can be used as such additional criteria.

Reply:
I agree that the feature is important for multiple reasons. We should explore ways to see this in action on staging tenants before with larger values.

The new value of 10 sounds safer, so I am content with that. Resolving.